licenserc.yaml

```diff
@@ -0,0 +1,12 @@
+header:
+  license:
+    spdx-id: MPL-2.0
+
+    content: |
+      This Source Code Form is subject to the terms of the Mozilla Public
+      License, v. 2.0. If a copy of the MPL was not distributed with this
+      file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  paths:
+    - '**/*.rs'
```

Comment:
I am... concerned that this isn't being respected for some reason, given that it's trying to check this file...